src/components/Amount.tsx

```diff
@@ -75,7 +75,7 @@ const Amount = ({
     if (value !== undefined) {
       amount = getAmount({ value, isFiat, decimals, nbOfDecimalsToShow, fullPrecision })
 
-      if (fadeDecimals && ['K', 'M', 'B', 'T'].some((char) => amount.endsWith(char))) {
+      if (fadeDecimals && MAGNITUDE_SYMBOL.some((char) => amount.endsWith(char))) {
```

Comment:
Does the fact that `MAGNITUDE_SYMBOL` includes an empty string (`''`) affect this in any way?

Reply:
Such a great catch. I think it may affect this, indeed. We should probably export only the actual symbols from the js-sdk.

Reply:
https://github.com/alephium/js-sdk/pull/158